scripts/common.py

```diff
@@ -45,7 +45,8 @@
 NODE_PATH = os.path.join(OPPIA_TOOLS_DIR, 'node-%s' % NODE_VERSION)
 NODE_MODULES_PATH = os.path.join(CURR_DIR, 'node_modules')
 FRONTEND_DIR = os.path.join(CURR_DIR, 'core', 'templates', 'dev', 'head')
-YARN_PATH = os.path.join(OPPIA_TOOLS_DIR, 'yarn-%s' % YARN_VERSION)
+# Assuming YARN_VERSION starts with a `v`
```

Comment:
`The following assumes that the YARN_VERSION starts with a 'v'.`

Reply:
Done.